docs/performance.asciidoc

```diff
@@ -0,0 +1,153 @@
+[[performance-tuning]]
+
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/nodejs/current/performance.html[elastic.co]
```

Comment:
I messed up when I moved some things around and now the actual generated html file is called `performance-tuning.html` because of the new anchor tag 😅 

**Option 1:** Rename the link to point to `performance-tuning.html`
**Option 2:** Keep the link as it is and rename the generated file back to just `performance.html`

I vote for option 1. But in that case we should also consider renaming the asciidoc file to match the generated html file. Just to make it easier to know what comes from what. All other doc files have a one-to-one filename correlation (`setup.asciidoc` being the only exception, as it generates `advanced-setup.html` - but we might consider "fixing" that as well).

What do you think?

Reply:
Option 1 sounds good to me.